cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/utils/MessageBatchEntityUtils.java

```diff
@@ -39,19 +43,19 @@ public class MessageBatchEntityUtils {
 
     public static SerializedEntity<MessageBatchEntity> toSerializedEntity(MessageBatchToStore batch,
                                                                           PageId pageId,
-                                                                          int maxUncompressedSize) throws IOException
-    {
+                                                                          CompressionType compressionType,
+                                                                          int maxUncompressedSize) throws IOException, CompressException {
```

Comment:
Here the formatting is also hard to read in a review. I think we need to try make code readable in code review as well as in IDE